Eatery Blue/Networking/EateryMemoryCache.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EateryMemoryCache.swift
+//  Eatery Blue
+//
+//  Created by Jayson Hahn on 10/25/23.
+//
+
+import Foundation
```

Comment:
Remove this import.

Reply:
I need foundation for date and TimeInterval objects

Reply:
Oh right I forgot.